reference/cf-k8s-controllers.yaml

```diff
@@ -1323,7 +1323,7 @@ subjects:
 ---
 apiVersion: v1
 data:
-  cf_k8s_controllers_config.yaml: '"kpackImageTag": "gcr.io/cf-relint-greengrass/cf-crd-staging-spike/"'
+  cf_k8s_controllers_config.yaml: 'kpackImageTag: gcr.io/cf-relint-greengrass/cf-k8s-controllers/kpack/beta'
```

Comment:
What is this reference directory used for?

Reply:
this directory will store runnable app images created as a result of kpack build.
I updated the path to reflect the codebase that is pushing images.